Xext/xres.c

```diff
@@ -304,12 +304,28 @@ ProcXResQueryClientResources(ClientPtr client)
 
     FindAllClientResources(resClient, ResFindAllRes, counts);
 
+    xXResType *scratch = calloc(sizeof(xXResType), lastResourceType + 1);
```

Comment:
The size should be the last argument. Not a big deal though.

Reply:
fixed. 
thanx